config/test.json

```diff
@@ -90,6 +90,7 @@
 		"reader/following-intro": true,
 		"reader/full-errors": true,
 		"reader/list-management": false,
+		"reader/new-post-notifications": false,
```

Comment:
might want it true here if we're going to write tests around it

Reply:
Yes! Good point.